plugins/plugin-site/src/components/PluginReleases.jsx

```diff
@@ -37,12 +37,15 @@ function PluginReleases({pluginId, versions}) {
     return (
         <div id="pluginReleases--container" className="container">
             {versions.map(version => {
-                const release = releases.find(release => version.version === release.tagName.replace(/^[^0-9]*/, '')) || {};
+                console.log(version.version);
+                // assume tags are in the form ${pluginId}${infix}${version} or ${prefix}${version}
+                // where pluginId may contain digits, but prefix and infix cannot and version always starts with a digit
+                const release = releases.find(release => version.version === release.tagName.replace(pluginId, '').replace(/^[^0-9]*/, '')) || {};
```

Comment:
Another readability one, this line got really long. Can you make something like a stripTag or something?